package.json

```diff
@@ -18,16 +15,11 @@
     "url": "https://github.com/mendixlabs/google-maps/issues"
   },
   "devDependencies": {
-    "@types/big.js": "0.0.31",
     "@types/core-js": "^0.9.34",
-    "@types/dojo": "^1.9.35",
     "@types/enzyme": "^2.4.36",
-    "@types/googlemaps": "^3.25.35",
     "@types/jasmine": "^2.5.35",
     "@types/karma": "^0.13.33",
     "@types/lodash": "^4.14.38",
```

Comment:
lodash removed
core-js is need because its being used in custom matchers see `matchers.ts`